src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -3723,8 +3723,12 @@ internal static bool AreRefsCompatibleForMethodConversion(RefKind x, RefKind y,
                     }
                     else
                     {
-                        badArguments = badArguments ?? ArrayBuilder<int>.GetInstance();
-                        badArguments.Add(argumentPosition);
+                        if (badArguments == default)
+                        {
+                            badArguments = BitVector.Create(paramCount);
```

Comment:
Yes, but it shouldn't harm since parameter count is expected to be small for most cases anyway. I didn't want to over complicate the code here to try to get the maximum bad argument index.

Reply:
Why not simply use ```argumentPosition + 1``` then?

Reply:
I missed that `BitVector` can expand later. I thought that this is just its size that can't grow later. It should indeed be possible to use `argumentPosition + 1`

Reply:
```suggestion
                            badArguments = BitVector.Create(argumentPosition + 1);
```